src/rendering/software/mod.rs

```diff
@@ -4,26 +4,14 @@
 
 #[cfg(all(
     test,
-    not(any(
-        // Without MMX the floating point calculations don't follow IEEE 754. So the
-        // tests fail to produce the correct checksums.
-        // FIXME: We use SSE as an approximation for the cfg, because MMX isn't
-        // supported by Rust just yet.
-        all(
-            target_arch = "x86",
-            not(target_feature = "sse"),
-        ),
-        // Additionally the x86_64-unknown-linux-musl target seems to be failing
-        // with lyon 0.15. I'd assume it's because of musl's libm doing
-        // something different than the glibc one, but I'm honestly not sure.
-        // FIXME: We likely want a perceptual image hash instead, which would be
-        // much more robust.
-        // https://github.com/LiveSplit/livesplit-core/issues/287
-        all(
-            target_os = "linux",
-            target_arch = "x86_64",
-            target_env = "musl",
-        ),
+
+    // Without MMX the floating point calculations don't follow IEEE 754. So the
+    // tests fail to produce the correct checksums.
+    // FIXME: We use SSE as an approximation for the cfg, because MMX isn't
+    // supported by Rust just yet.
+    not(all(
+        target_arch = "x86",
+        not(target_feature = "sse"),
```

Comment:
If the image hash is good enough, we shouldn't need this anymore either.

Reply:
Seems like some of the image comparisons are still slightly off. Could we retrieve the images from the targets that are failing and see what it actually looks like?

Reply:
You could try to run these tests locally. Just install the specific target and run the tests with the target.